packages/ethereum-storage/test/smartcontract-manager.test.ts

```diff
@@ -250,6 +252,11 @@ describe('SmartContractManager', () => {
   });
 
   it('cannot add hash to ethereum if block of the transaction is not fetchable within 23 confirmation', async () => {
+    // fake the creation of new blocks on ethereum
+    const blockInterval = setInterval(async () => {
```

Comment:
Would the node stop working on local networks if it's not running this test?

Reply:
the node locally running with Ganache works fine 👍